packages/sw-background-sync-queue/build.js

```diff
@@ -26,8 +26,29 @@ const mainModuleBuilds = generateBuildConfigs({
 }, __dirname, 'goog.backgroundSyncQueue').map(buildJSBundle);
 
 module.exports = () => Promise.all([
-  ...mainModuleBuilds,
 
+  ...mainModuleBuilds,
```

Comment:
@jeffposnick does this match what you have done for the other projects in terms of build process?